stories/__tests__/SQFormButton.stories.test.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {composeStories} from '@storybook/testing-react';
+import * as stories from '../SQFormButton.stories';
+
+const {
+  Default: SQFormButton,
+  WithTestField: SQFormButtonWithField
+} = composeStories(stories);
+
+describe('Default Tests', () => {
+  it('should render a button with text', () => {
+    render(<SQFormButton />);
+
+    const submitButton = screen.getByRole('button', {name: /form submission/i});
+
+    expect(submitButton).toBeInTheDocument();
+    expect(submitButton).toHaveTextContent('Submit');
+  });
+
+  it('should render a reset button given the type', () => {
+    render(<SQFormButton type="reset" />);
+
+    const resetButton = screen.getByRole('button', {name: /form submission/i});
```

Comment:
If the button is a reset button, I wouldn't expect the name to be `form submission` 🤔

Reply:
That's the default title in SQFormButton. I can pass in an override to make it say "Reset Form" or something similar.

Reply:
Sounds like an improvement that could be made to the component itself

Reply:
> That's the default title in SQFormButton. I can pass in an override to make it say "Reset Form" or something similar.

Is this a bug though? (I did this LOL)

If the `type` prop is set to `RESET`, the title should change to Reset Form

Reply:
created issue #220 for this